empress/histogram/histogram_display.py

```diff
@@ -9,6 +9,11 @@ def plot_histogram_to_ax(ax: plt.Axes, histogram):
     ax.ticklabel_format(style="sci", axis="y", scilimits=(0, 0))
     ax.set_ylabel("Number of MPR Pairs")
 
+def plot_histogram_to_ax(ax: plt.Axes, histogram):
```

Comment:
Duplicate -- it seems like these duplicates might have been created my merge? Please be a little careful when you merge.

Reply:
Just reset this file to the one in master -- https://stackoverflow.com/questions/215718/how-can-i-reset-or-revert-a-file-to-a-specific-revision

Reply:
Ok!